### DISCUSSION TITLE
--Mavericks --

### DISCUSSION THREAD

**Discussion by RyaanHassanKhan** [RyaanHassanKhan/C-programs-#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RyaanHassanKhan** [comment]
c-programms

**Comment by khanabrose** [comment]
hhhhhh

**Comment by RyaanHassanKhan** [comment]
Make one folder for all the codes, under a repository -c programmms

**Reply by khanabrose** [reply]
this is good for me.......

**Reply by RyaanHassanKhan** [reply]
Check my repo
